packages/google_identity_services_web/README.md

```diff
@@ -0,0 +1,85 @@
+# google_identity_services_web
+
+A JS-interop layer for Google Identity's Sign In With Google SDK.
+
+See the original JS SDK reference:
+
+* [Sign In With Google](https://developers.google.com/identity/gsi/web)
+
+## Usage
+
+This package is the Dart JS-interop layer of the new **Sign In With Google**
+SDK. Here's the API references for both of the sub-libraries:
+
+* `id.dart`: [Sign In With Google JavaScript API reference](https://developers.google.com/identity/gsi/web/reference/js-reference)
+* `oauth2.dart`: [Google 3P Authorization JavaScript Library for websites - API reference](https://developers.google.com/identity/oauth2/web/reference/js-reference)
+* `loader.dart`: An (optional) loader mechanism that installs the library and
+resolves a `Future<void>` when it's ready.
+
+### Loading the SDK
+
+There are two ways to load the JS SDK in your app.
+
+#### Modify your index.html (most performant)
+
+The most performant way is to modify your `web/index.html` file to insert a
+script tag [as recommended](https://developers.google.com/identity/gsi/web/guides/client-library).
+Place the `script` tag in the `<head>` of your site, next to the script tag that
+loads `flutter.js`, so the browser can downloaded both in parallel:
+
+```html
+<!-- Your index.html -->
+<head>
+    ...
+    <script src="https://accounts.google.com/gsi/client" async defer></script>
+    <!-- ^^^^^^^-- add the new script there -->
```

Comment:
Using `--` inside an HTML comment other than for the `-->` is actually invalid HTML, so that should be removed.

Reply:
Yikes! TIL

Reply:
I moved all of this to code excerpts